src/models/goalSimilarityGroup.js

```diff
@@ -0,0 +1,55 @@
+const {
+  Model,
+} = require('sequelize');
+
+/**
+ *
+ * @param {} sequelize
+ * @param {*} DataTypes
+ */
+export default (sequelize, DataTypes) => {
+  class GoalSimilarityGroup extends Model {
+    static associate(models) {
+      GoalSimilarityGroup.belongsTo(models.Recipient, { foreignKey: 'recipientId', as: 'recipient' });
+      GoalSimilarityGroup.hasMany(models.GoalSimilarityGroupGoal, { foreignKey: 'goalSimilarityGroupId', as: 'goalSimilarityGroups' });
+      GoalSimilarityGroup.belongsToMany(models.Goal, {
+        through: models.GoalSimilarityGroupGoal,
+        foreignKey: 'goalSimilarityGroupId',
+        otherKey: 'goalId',
+        as: 'goals',
+      });
```

Comment:
Can we relocate these to the goalSimilarityGroupGoal.js file. In my efforts I have found it prevents possible race conditions to have all associations defined in the file for the model imposing the associations. the only change that will be required is adding "models." before the "GoalSimilarityGroup"